src/index.html

```diff
@@ -32,15 +32,15 @@
     content="https://github.com/chryz-hub/chryz-hub.github.io/blob/master/public/myocatact1%20(1).svg" />
 
   <!-- FAVICON -->
-  <link rel="icon" type="image/ico" href="./src/assets/images/favicon.ico" />
+  <link rel="icon" type="image/ico" href="./src/assets/favicon.ico" />
```

Comment:
```suggestion
  <link rel="icon" type="image/ico" href="./assets/favicon.ico" />
```
I think the src is not needed due the index html is in src